src/requirements.cpp

```diff
@@ -125,7 +125,11 @@ std::string item_comp::to_string( const int batch, const int avail ) const
     const int c = std::abs( count ) * batch;
     const auto type_ptr = item::find_type( type );
     if( type_ptr->count_by_charges() ) {
-        if( avail > 0 ) {
+        if( avail == INT32_MAX ) {
+            return string_format( npgettext( "requirement", "%2$d %1$s of infinite", "%2$d %1$s of infinite",
```

Comment:
```suggestion
            //~ %1$s: item name, %2$d: charge requirement
            return string_format( npgettext( "requirement", "%1$s (%2$d of infinite)", "%1$s (%2$d of infinite)",
```
To be consistent with the code below.